quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/configuration/test/AbstractConfigurationTest.java

```diff
@@ -172,6 +173,19 @@ protected void assertExternalConfig(Map<String, String> expectedValues) {
         expectedValues.forEach(this::assertExternalConfig);
     }
 
+    protected void assertConfigNull(String key, boolean isExternal) {
+        Function<String, ConfigValue> getConfig = isExternal ? Configuration::getConfigValue : Configuration::getKcConfigValue;
+        assertThat(String.format("We expect that the value is null for key '%s'", key), getConfig.apply(key).getValue(), nullValue());
+    }
+
+    protected void assertConfigNull(String key) {
+        assertConfigNull(key, false);
+    }
+
+    protected void assertExternalConfigNull(String key) {
+        assertConfigNull(key, true);
+    }
```

Comment:
@shawkins The only thing that was added to the 26.2.x release. I could have used `assertNull()` in the code, but I decided to rather provide the same concept as for main/26.3 to not having so many differences. Is it ok for you?

Reply:
It's fine with me.